### ISSUE TITLE
Advance feature: showing color picker on "fill()" and "stroke()", and preview clolor

### ISSUE THREAD

**Issue by yining1023** [processing/p5.js-web-editor#90]
something like this on https://thimble.mozilla.org
<img width="536" alt="screen shot 2016-09-07 at 1 29 35 pm" src="https://cloud.githubusercontent.com/assets/8662372/18322160/3f229750-74ff-11e6-8c4c-39ce48401797.png">

<img width="324" alt="screen shot 2016-09-07 at 1 31 34 pm" src="https://cloud.githubusercontent.com/assets/8662372/18322202/723a1a14-74ff-11e6-9a95-4c1ab8c41a8f.png">

**Comment by ankur54** [comment]
Hi. I would like to work on this issue. I am thinking of using jQuery color picker plugins like jQuery Color Picker or Drupal Color Picker etc. so that on hovering over the color code, the color picker appears. Is that all you want?

**Comment by catarak** [comment]
i don't want to add a jquery dependency to this project. honestly, i think this issue is pretty low priority, so unless you feel strongly that it should be in the web editor, i recommend picking up another issue.

**Comment by ankur54** [comment]
To be honest I think, this feature would be the a bit cooler to have, as we can dynamically select color just by adjusting the pointer. Thats just my opinion though

**Comment by RasheedAZ** [comment]
I think this is a _necessary_ feature to have bearing in mind p5.js website's opening statement:

> p5.js is a JavaScript library for creative coding, with a focus on making coding accessible and inclusive for artists, designers, educators, **_beginners_**, and anyone else!

Hop you guys can assign it high priority or medium at least. 

Thanks :)

**Comment by catarak** [comment]
I hear you! I think there are pros and cons. When I first started working on this project, it was hard to know what was important to add and what wasn't. There's a part of me that's like, yeah, it would be awesome to make choosing colors for your sketch easier! But also, there's other stuff to keep in mind:
- One thing that's cool about p5.js is that it introduces folks to thinking about stuff like colors, images, sound, and so on, as numbers. Having the experience of translating colors into numbers in that way is perhaps useful, from a pedagogical standpoint. 
- One of the guiding principles of the web editor "simplicity". (see the statement on the [web editor readme](https://github.com/processing/p5.js-web-editor#p5js-web-editor)).  To get at this, I try to ask questions like, "does this add something that doesn't exist elsewhere?" For me, I'm happy to use an external color picker, but I'm not every user.

In summary, I'm down to talk and think about this more, but right now, I'm not 100% convinced this is a major feature.

**Comment by catarak** [comment]
I've changed my mind about this issue and think it would add a lot to the editor! So the issue is more visible, I'm going to close this one in favor of #1867.
